src/app/connections/create-page/review/review.component.ts

```diff
@@ -13,12 +14,14 @@ import { CanComponentDeactivate } from '../../../common/can-deactivate-guard.ser
 import { ModalService } from '../../../common/modal/modal.service';
 import { ConnectionService } from '../../../store/connection/connection.service';
 import { log, getCategory } from '../../../logging';
+import { UserService } from '../../../common/user.service';
 
 const category = getCategory('Connections');
 
 @Component({
   selector: 'syndesis-connections-review',
   templateUrl: 'review.component.html',
+  styles: [ '' ],
```

Comment:
I don't think this line is necessary

Reply:
It's not, but will make its way to the transpiled app unless removed. As a rule of thumb, `styles` or `styleUrls` properties should be removed from the decorator hash if not in use.

BTW, the new linter will complain when arrays containing stuff with blank spaces around. Is everybody agree?

Reply:
Not sure why this is here tbh, I must have been testing things out. Sorry.

Reply:
Sure @deeleman sounds good to me.